### DISCUSSION TITLE
Does a `backoffLimit` of 0 disable backoff?

### DISCUSSION THREAD

**Discussion by jensbodal** [sindresorhus/got#2309] [Q&A] [answered]
We are trying to retry without a backoff on some specific calls.  We have set `backoffLimit` to 0 and are seeing some weird results with gaps between retries. Just want to confirm setting the `backoffLimit` to `0` disables exponential backoff so can investigate elsewhere. Thanks!

**Comment by sindresorhus** [comment, marked as answer]
The `backoffLimit` parameter sets an upper limit on the `computedValue` for the delay between retries, not the behavior of the exponential backoff. To disable exponential backoff, set `calculateDelay` function to return a fixed value, like `return 1000;` in the `retry` object. This way, the delay between retries remains constant, achieving the desired effect.

**Comment by jensbodal** [comment]
So if I just return a fixed value I saw infinite retries. I had to add a breakpoint based on the number of retries as well.  Maybe this was obvious but I missed it :) 

```
        calculateDelay: (retryObject) => {
          const { attemptCount } = retryObject;

          if (attemptCount >= RETRY_LIMIT) {
            return 0;
          }
          return 100;
        },
```
